tests/basic_server.py

```
@@ -0,0 +1,111 @@
+            print(f"An error occurred: {e}")
+            close_connection(client)
+            break
+
+
+# Receiving / Listening Function
+def receive():
+    while True:
+        try:
+            # Accept Connection
+            client, address = server.accept()
+            print(f"Connected with {str(address)}")
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)